### PR TITLE
Implemented further changes and tested them

### PR DESCRIPTION
**Pull request by shravan-cc**

### CODE REVIEW

File: assignment.js [hunk 1]
```diff
@@ -315,17 +314,255 @@ export function employeeDataProcessor() {
...
+}
+
+const generateArray = (number) => {
+  const array = [];
+  for (let i = 1; i <= number; i++) {
+    array.push(i);
+  }
+  return array;
+};
+const generateObject = (generateArray) => {
```
Comment (shripada) [inline assignment.js]: generateArray name sounds like a function...

File: assignment.js [hunk 2]
```diff
@@ -315,17 +314,255 @@ export function employeeDataProcessor() {
...
+  const obj = {};
+  obj.odd = generateArray.filter((number) => number % 2 !== 0);
+  obj.even = generateArray.filter((number) => number % 2 === 0);
+  return obj;
+};
+
+const transformObject = (generateObject) => {
```
Comment (shripada) [inline assignment.js]: same issue with generateObject

**Review by shripada — CHANGES_REQUESTED** [review]

File: assignment.js (hunk 1 above)
Comment (shravan-cc) [reply assignment.js]: Made the neccassary changes.

File: assignment.js (hunk 2 above)
Comment (shravan-cc) [reply assignment.js]: Made the neccassary changes.

**Review by shripada — APPROVED** [review]

**Pull request merged**